### ISSUE TITLE
Large IPA file size - iOS

### ISSUE THREAD

**Issue by anastely** [Agontuk/react-native-geolocation-service#245]
Hey!
First of all Thanks for this package, I have an issue with it on iOS.

As you know in the setup we add swift and bridging files, those make a size issue 

So after deleting this package and related stuff (swift & bridging files) the IPA size down from 172 to 61 MB

I've opened this up to consider it.
Regards

**Comment by Agontuk** [comment]
Yes, using swift framework increases app size, but it shouldn't be this much as far as I know. There should be an option to strip swift symbols during archiving, can you check if it can reduce the size ? I'm not currently working in react-native projects, so can't test this.

**Comment by ravirajn22** [comment]
@anastely Apple will only bundle swift core library on older versions of iOS and also only required files. The true size of the app downloaded can be found in TestFlight. The IPA file contains code to support all CPU architectures so will be big.
